package.json

```diff
@@ -182,6 +182,7 @@
     "@types/validator": "^9.4.2",
     "@typescript-eslint/parser": "^3.10.1",
     "abortcontroller-polyfill": "1.3.0",
+    "astral-regex": "^2.0.0",
```

Comment:
is it needed?

Reply:
Without this add tests keep failing, we can try a workaround together?